src/main/java/io/paradoxical/cassandra/loader/DbScriptsRunner.java

```diff
@@ -43,7 +43,8 @@ public void run(Session rootSession) throws Exception {
         if (dbRunnerConfig.getCreateKeyspace() != null && dbRunnerConfig.getCreateKeyspace()) {
             try {
                 rootSession.execute("CREATE KEYSPACE " + dbRunnerConfig.getKeyspace() + "\n" +
-                                    "WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
+                                    "WITH REPLICATION = { 'class' : '" + dbRunnerConfig.getReplicationStrategy() +
+                                    "', 'replication_factor' : " + dbRunnerConfig.getReplicationFactor() +") };");
```

Comment:
replication factor only works with replication strategy simple... so its a little weird to make the strategy configurable.

Also we should add a `IF NOT EXISTS` after the keyspace name to prevent the error we try and catch.